pycaret/internal/preprocess/time_series/forecasting/preprocessor.py

```diff
@@ -112,6 +114,101 @@ def _add_imputation_steps(
+        ):
+            # self._add_limitation_steps(
+            #     limits=limit_exogenous, target=False
+            self.logger.warning("Applying limits to exogenous variables is not yet implemented.")
+            
+
+    def _add_limitation_steps(self,
+                              limits: List[Union[int,float,None]],
+                              target: bool = True):
+        """Limit/scale Possible forecast values using sktime's ScaledLogitTransformer
+
+        Example:
+        --------
+        >>> limits = None # default - no limits
+        >>> limits = [0, 10000000] # lower and upper limit
+        >>> limits = [0, None]  # lower limit only
+        >>> limits = [None, 10000000] # upper limit only
+
+        Parameters
+        ----------
+        limits : List[Union[int,float,None]]
+            A list (of two values) of the minimum and maximum values
+            
+        target : bool, optional
+            If True, limit is added to the target variable steps
+            If False, limit is added to the exogenous variable steps,
+            (not implemented yet)
+            by default True
+
+        Raises
+        ------
+        TypeError
+            (1)  value in `limits` is not None or float or int.
+            (2) `limits` is not a list.
+        ValueError
+            (1) `limits` is a list of length not equal to 2.
+        NotImplementedError
+            (1) `limits` is a subclass of BaseEstimator.
+            (2) `target` is False, as exogenous limiting is not implemented yet.
+        """
+
+        type_ = "Target" if target else "Exogenous"
+        self.logger.info(f"Setting up limits for {type_} variable(s).")
+        if isinstance(limits, list):
+            if len(limits) == 2:
+                # Valid limits length
+                for i in limits:
+                    if not (isinstance(i, (int, float)) or i is None):
+                        raise TypeError(
+                            f"{type_} limit value {i}, '{type(i)}' is not of allowed type.")
+                if all([i is None for i in limits]):
+                    # No limits as both is none, but exit silently
+                    return
+                # create limiter
+                limiter = ScaledLogitTransformer(*limits)
+
+            else:
+                raise ValueError(f"{type_} limits list must be of length 2.")
+
+        elif issubclass(limits, BaseEstimator):
+            # TODO: Implement passing sktime compatible transformer directly.
+            raise NotImplementedError(
+                "Using transformers directly is not yet implemented,\
+                    please use numeric limits only for now.")
+        else:
+            raise TypeError(
+                f"{type_} forecast limit Type '{type(limits)}' is not of allowed type."
+            )
+
+        if target:
+            print("creating pipeline limit step")
+            self.pipe_steps_target.extend([("target_limiter", limiter)])
```

Comment:
I would just call it limiter. No need to explicitly call out `target` since this will go inside `TransformedTargetForecaster` and will be different from the steps that go into the `ForecastingPipeline` for exogenous variables.